daemon/daemon_unix.go

```diff
@@ -419,6 +419,7 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 		netlabel.DriverMTU:        strconv.Itoa(config.Mtu),
 		bridge.EnableIPMasquerade: strconv.FormatBool(config.Bridge.EnableIPMasq),
 		bridge.EnableICC:          strconv.FormatBool(config.Bridge.InterContainerCommunication),
+		netlabel.EnableIPv6:       strconv.FormatBool(config.Bridge.EnableIPv6),
```

Comment:
I don't understand the distinction. Unless I've misread the libnetwork code, it all ends up as properties on the network struct...so I'm not sure what it means to be 'driver specific'.

Regardless, this is just making the default bridge construction be more similar to manually created networks (for consistency).